include/numerics/type_vector.h

```diff
@@ -311,8 +311,9 @@ class TypeVector
                              Real tol = TOLERANCE) const;
 
   /**
-   * @returns \p true iff two vectors occupy approximately the same
-   * physical location in space, to within an absolute tolerance of \p TOLERANCE.
+   * @returns \p true iff two vectors occupy the same physical location in space,
+   * to within tolerance of the type T. For floating point types, the function
```

Comment:
I maybe would have said "calls operator== on every component" of the Point, but this is definitely an improvement over what was there previously.